.github/workflows/lint-pr.yml

```diff
@@ -8,7 +8,6 @@ jobs:
   pr-title-lint:
     runs-on: ubuntu-latest
     permissions:
-      contents: read
```

Comment:
I guess something more is needed here, 'cause I still need to do the edit/save/edit/save drill almost every time I push a new commit.

Reply:
I have a feeling it's something in the calling workflow. Which is not permission-related. I am working on it now.

Reply:
In other news: Did you get my review request that was for statnett/k3a, or must I assign explicitly?